feeds/wlan-ap/opensync/src/platform/openwrt/src/lib/target/src/radio.c

```diff
@@ -669,6 +674,150 @@ static void callback_Hotspot20_Icon_Config(ovsdb_update_monitor_t *mon,
 
 }
 
+enum {
+	WIF_APC_ENABLE,
+	__WIF_APC_MAX,
+};
+
+static const struct blobmsg_policy apc_enpolicy[__WIF_APC_MAX] = {
+		[WIF_APC_ENABLE] = { .name = "enabled", BLOBMSG_TYPE_BOOL },
+};
+
+const struct uci_blob_param_list apc_param = {
+	.n_params = __WIF_APC_MAX,
+	.params = apc_enpolicy,
+};
+
+void APC_config_update(struct schema_APC_Config *conf)
+{
+	struct uci_package *apc;
+	struct uci_context *uci;
+	struct blob_buf apcb = { };
+
+	LOGN("APC: APC_config_update");
+	uci = uci_alloc_context();
+
+	uci_load(uci, "apc", &apc);
+	blob_buf_init(&apcb, 0);
+
+	if (conf->enabled == true) {
```

Comment:
This is the only parameter in the table, so this function should only get called if enabled has changed state.  It still feels weird not to check for a change, because if we extend the table in the future it may create bug.

What happens if two instances get started?

Reply:
Two instances cannot get started the /etc/init.d/apc start doesn't start a new instance if already an instance is running.